docs/developing.md

```diff
@@ -159,10 +161,11 @@ make run-operator
 
 #### Run Hive Operator Using Custom Images
 
- 1. Build and publish a custom Hive image from your current working dir: `$ IMG=quay.io/{username}/hive:latest make buildah-dev-push`
- 2. Deploy with your custom image: `$ DEPLOY_IMAGE=quay.io/{username}/hive:latest make deploy`
- 3. After code changes you need to rebuild the Hive images as mentioned in step 1.
- 4. Delete the running Hive pods using following command, so that the new pods will be running using the latest images built in the previous step.
+ 1. Build a custom Hive image from your current working dir: `$ IMG=quay.io/{username}/hive:latest make image-hive`
```

Comment:
any particular reason to have every step listed as step number `1.`. It works AFAICT, but the rest of the docs don't use this numbering system.

Reply:
If everything is number `1.`, then other lines do not need to be updated when lines are added or removed.